erikiado/jp2_online
# Task/carga preguntas

#### What's this PR do?

Creates a script to load all the questions in the database.

#### Where should the reviewer start?

estudios_socioeconomicos/load.py
estudios_socioeconomicos/preguntas.pkl

#### How should this be manually tested?

try running it and check in the admin if the questions are saved.

to run, open the django shell:
python manage.py shell

then import the function and run it:
from estudios_socioeconomicos.load import load_data
load_data()


This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
